transformers/shared/constants/method.go

```diff
@@ -83,19 +84,22 @@ func VatABI() string  { return getContractABI("MCD_VAT") }
 func VowABI() string  { return getContractABI("MCD_VOW") }
 
 func auctionFileMethod() string { return getSolidityFunctionSignature(FlipABI(), "file") }
-func biteMethod() string        { return getSolidityFunctionSignature(CatABI(), "Bite") }
+func biteMethod() string        { return getSolidityFunctionSignature(Cat110ABI(), "Bite") }
+func catFileBoxMethod() string {
+	return getOverloadedFunctionSignature(Cat110ABI(), "file", []string{"bytes32", "uint256"})
+}
 func catFileChopLumpMethod() string {
-	return getOverloadedFunctionSignature(CatABI(), "file", []string{"bytes32", "bytes32", "uint256"})
+	return getOverloadedFunctionSignature(Cat100ABI(), "file", []string{"bytes32", "bytes32", "uint256"})
```

Comment:
Just double checking that this is right - this is Cat100 not 110 because there is no chop lump in the new contract correct?

Reply:
Yep, that's correct. Though, I think that Cat110 has a file method that has the same signature, so this method could probably be renamed to `catFileChopDunkMethod()` as part of the VDB-1632 work.